src/app/api/services/dot-ui-colors/dot-ui-colors.service.ts

```diff
@@ -42,32 +43,32 @@ export class DotUiColorsService {
         return `${rgb.r}, ${rgb.g}, ${rgb.b}`;
```

Comment:
the values into this.currentColors and window.getComputedStyle don't should be the same?

Reply:
No necessarily, `this.currentColors` is updated when you call `setColors` with a color param, then we applied to the and then you have the same, so at some point you don't have the same.